Gemfile

```diff
@@ -1,5 +1,5 @@
 source 'https://rubygems.org'
-ruby "2.4.1"
+ruby "2.5.3"
```

Comment:
This is a serious change. Denis have already tried updating to this version and faced some compatibility problems. We need to examine the whole codebase carefully and test everything to make sure this won't break things.